lib/record_store/record/aaaa.rb

```diff
@@ -26,5 +30,11 @@ def valid_address?
     rescue IPAddr::InvalidAddressError
       errors.add(:address, 'is invalid')
     end
+
+    def normalize_address(value)
```

Comment:
Does normalize_address something we will want to call for all record types somehow? I was wondering if we could define it in the parent class Record, then override if necessary in the children classes?

Reply:
Probably a good future improvement